gemup.js

```diff
@@ -7,6 +8,7 @@
       acl: 'public-read',
       app: 'force',
       geminiHost: 'https://media.artsy.net',
+      storeWithOriginalFilename: false,
```

Comment:
@ansor4 my instinct would be to set this to `true` by default because that's been gemup's behavior in the past. Switching to using UUID-based filenames definitely feels like a win, but I don't feel like making it the default is worth turning it from a non-breaking change to a breaking change.

The main thing that I could see that change impacting would be Positron - I'm not familiar enough with it to know if it relies on the filename in any way, but it feels a bit risky to me.

Reply:
> my instinct would be to set this to true by default 

I think that's a good call to preserve backwards-compatibility and offer this new behavior as opt-in rather than opt-out.

Maybe we can rename the arg as `preserveOriginalFilename` and default to `false`? When `true`, replace the filename with a UUID?

Reply:
hmm sounds good, i was thinking (but didn't actually follow through) of bumping the version as well so other repos would just reference v0.0.3 and be fine. But maybe we can switch the default value to `true` on version bump later.

Reply:
It depends on how those repos have the version set. If we make a breaking change and only bumped to version to v0.0.4, a `yarn update` might happily bump to version from `v0.0.3` to v0.0.4`.

So either we can make this change safely backwards-compatible (I think this is preferred) and update this Viewing Room-specific usage to trigger the new behavior. Or we release a new major version `v1.0.0` of gemup. Maybe we do both eventually :man_shrugging: